src/Tribe/REST/V1/Post_Repository.php

```diff
@@ -88,8 +88,15 @@ public function get_data( $id, $context = '' ) {
 
 	/**
 	 * {@inheritdoc}
+	 *
+	 * @since TBD Returns 401 Unauthorized if Event Tickets Plus is not loaded.
 	 */
 	public function get_attendee_data( $attendee_id, $context = 'default' ) {
+		// Early bail: ET Plus must be active to get Attendee data.
+		if ( ! class_exists( "Tribe__Tickets_Plus__Main" ) ) {
```

Comment:
:no_entry_sign: String "Tribe__Tickets_Plus__Main" does not require double quotes; use single quotes instead

<sub>hash: 5e03be83d772af3968ca36859e4ac6b4</sub>